bandit/formatters/text.py

```diff
@@ -27,6 +27,8 @@
     >> Issue: [B301:blacklist_calls] Use of unsafe yaml load. Allows
        instantiation of arbitrary objects. Consider yaml.safe_load().
 
+       More Info: https://bandit.readthedocs.io/en/latest/
```

Comment:
Can you move the more info below the location?  That would be more consistent with other formatters.

Reply:
done